sympy/matrices/dense.py

```diff
@@ -1570,12 +1570,21 @@ def casoratian(seqs, n, zero=True):
     return Matrix(k, k, f).det()
 
 
-def randMatrix(r, c=None, min=0, max=99, seed=None, symmetric=False, percent=100):
+def randMatrix(r, c=None, min=0, max=99, seed=None, prng=None, symmetric=False, percent=100):
     """Create random matrix with dimensions ``r`` x ``c``. If ``c`` is omitted
     the matrix will be square. If ``symmetric`` is True the matrix must be
     square. If ``percent`` is less than 100 then only approximately the given
     percentage of elements will be non-zero.
 
+    The pseudo-random number generator used to generate matrix is chosen in the
+    following way:
+
+    * if ``prng`` is supplied, it will be used as random number generator;
```

Comment:
You should specify that it has to be a `random.Random` class (or at least have the same API).